static/client/common.js

```diff
@@ -299,3 +299,53 @@ CCC.Common.escapeSpaces = function(text) {
       .replace(/  /g, '\u00A0 ').replace(/  /g, '\u00A0 ')
       .replace(/^ /gm, '\u00A0');
 };
+
+/**
+ * Detect URLs in the provided document fragment and replace with links.
+ * @param {!Element} el A DOM element to scan and modify.
+ */
+CCC.Common.autoHyperlink = function(el) {
+  var isSvg = el.namespaceURI == CCC.Common.NS;
+  var children = el.childNodes;  // This is a live NodeList.
+  for (var i = children.length - 1, child; (child = children[i]); i--) {
```

Comment:
Presumably you are processing this in reverse order because `children` is live.  But since neither `i` nor `children` are otherwise referenced, why not
```JS
  for (child of el.childNodes) {
```
(or, if necessary, `for (child of Array.from(el.childNodes))`)?

Reply:
Changed in 44 places, but not this one.